src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -110,6 +123,10 @@ public DashboardWebApplication(Action<WebApplicationBuilder>? configureBuilder =
 
         _app = builder.Build();
 
+        _dashboardOptionsMonitor = _app.Services.GetRequiredService<IOptionsMonitor<DashboardOptions>>();
+        // Get options to run validation during startup.
+        _ = _dashboardOptionsMonitor.CurrentValue;
```

Comment:
There's a method for this https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.optionsbuilderextensions.validateonstart?view=dotnet-plat-ext-8.0